digiroad2-geo/src/main/scala/fi/liikennevirasto/digiroad2/lane/Lane.scala

```diff
@@ -179,9 +179,12 @@ object LaneType {
   case object MassTransitTaxi extends LaneType { def value = 9; def typeDescription = "Mass transit or taxi lane"; def finnishDescription = "Joukkoliikenteen kaista / taksikaista"; }
   case object Truckway extends LaneType { def value = 10; def typeDescription = "Truckway"; def finnishDescription = "Raskaan liikenteen kaista"; }
   case object Reversible extends LaneType { def value = 11; def typeDescription = "Reversible lane"; def finnishDescription = "Vaihtuvasuuntainen kaista"; }
-  case object Combined extends LaneType { def value = 20; def typeDescription = "Combined walking and cycling lane"; def finnishDescription = "Yhdistetty jalankulun ja pyöräilyn kaista"; }
-  case object Walking extends LaneType { def value = 21; def typeDescription = "Walking lane"; def finnishDescription = "Jalankulun kaista"; }
-  case object Cycling extends LaneType { def value = 22; def typeDescription = "Cycling lane"; def finnishDescription = "Pyöräilykaista"; }
+  case object BicycleLane extends LaneType { def value = 12; def typeDescription = "Bicycle lane"; def finnishDescription = "Pyöräkaista"; }
+  case object Combined extends LaneType { def value = 20; def typeDescription = "Combined bike path and sidewalk"; def finnishDescription = "Yhdistetty pyörätie ja jalkakäytävä"; }
```

Comment:
pyorätiellä on nyt kaksi toisistaan poikkeavaa englannistusta. Valitaan jompikumpi.

Reply:
Löyty nuo termit Description of data objects dokkarista:
https://vayla.fi/documents/25230764/0/Digiroad+description+of+data+objects+3_2021.pdf/a2dfe521-8de7-9759-add8-f24ec37e2709/Digiroad+description+of+data+objects+3_2021.pdf?t=1623857465592

Mennäänkö samoilla vai muutetaanko?

Reply:
No olkoon sitten, jos on virallisesti päätetty noin =)

Reply:
Jaa, jos näin on jossakin päätetty ok, saattaa kyl tuotta kivaa hämäänystä x vuoden päästä. Ei sillee sanotaan vaan että dokumentaatio sanoi näin aikoinaan.